sklearn/feature_selection/_rfe.py

```diff
@@ -478,6 +545,29 @@ def _more_tags(self):
 
         return tags
 
+    def get_metadata_routing(self):
+        """Get metadata routing of this object.
+
+        Please check :ref:`User Guide <metadata_routing>` on how the routing
+        mechanism works.
+
+        .. versionadded:: 1.6
+
+        Returns
+        -------
+        routing : MetadataRouter
+            A :class:`~sklearn.utils.metadata_routing.MetadataRouter` encapsulating
+            routing information.
+        """
+        router = MetadataRouter(owner=self.__class__.__name__).add(
+            estimator=self.estimator,
+            method_mapping=MethodMapping()
+            .add(caller="fit", callee="fit")
+            .add(caller="predict", callee="predict")
+            .add(caller="score", callee="score"),
+        )
+        return router
+
 
 class RFECV(RFE):
```

Comment:
Side note: We inherit from RFE and we create RFE object inside the class. I have the impression that this would be such an anti-pattern :)

Reply:
RFECV is such an odd estimator. We also do things like 

```py
        rfe.fit(X, y)

        # Set final attributes
        ...
        self.estimator_ = clone(self.estimator)
        self.estimator_.fit(self._transform(X), y)
```

I'm like, WUT? How many times do we need to fit the same thing.